sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Models/QueryRequestOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// Optional parameters to use when querying for Time Series Insights.
+    /// </summary>
+    public abstract class QueryRequestOptions
+    {
+        /// <summary>
+        /// For the environments with warm store enabled, the query can be executed either on the 'WarmStore' or 'ColdStore'.
+        /// This parameter in the query defines which store the query should be executed on. If not defined, the query
+        /// will be executed on the cold store.
+        /// </summary>
+        public StoreType StoreType { get; set; }
+
+        /// <summary>
+        /// A Time series expression (TSX) filter written as a single string.
```

Comment:
Capitalize proper product name Time Series

Not sure about TSX as the document title says Time Series expression in the menu, but Time Series Expression in the document.

Reply:
I looked at how they do it in ms docs. 

When referring to it right before the acrynom, it's all upper case:
Time Series Expression (TSX)

When referring to it without the acrynom, it's lower case expression:
Time Series expression

That format is pretty consistent across the documentation. I will change the format to match the documentation.

Good catch!